presto-native-execution/etc/node.properties

```diff
@@ -1,4 +1,4 @@
-node.environment=testing
```

Comment:
Would this change require updating README on how to run e2e tests?

Reply:
Good catch! No change is required in the README. But we do have to change the value of `node.environment` inside `HiveExternalWorkerQueryRunner.java`

Reply:
I will make this change.